capa/features/extractors/elffile.py

```diff
@@ -85,8 +105,7 @@ def extract_file_features(elf: ELFFile, buf: bytes) -> Iterator[Tuple[Feature, i
 
 
 FILE_HANDLERS = (
-    # TODO(williballenthin): implement extract_file_export_names
-    # https://github.com/mandiant/capa/issues/1607
+    extract_file_export_names,
```

Comment:
would you please add a few test cases that demonstrate the new import/export handling working as expected?

Reply:
that works. or you can spot check a few.